seshdash/views.py

```diff
@@ -267,11 +267,11 @@ def get_high_chart_data():
 
     # Getting climat conditions
 
-     high_cloud_cover = Site_Weather_Data.objects.filter(date__range=[five_day_past2,five_day_future2]).values_list('cloud_cover', flat=True).order_by('date')
+     high_cloud_cover = list(Site_Weather_Data.objects.filter(date__range=[five_day_past2,five_day_future2]).values_list('cloud_cover', flat=True).order_by('date'))
```

Comment:
Use also need to filter by site location. There can be multiple sites at different locations. Currently in your filter there is nothing that will limit the location.